libs/entry-components/styles/modules/components/toggle/_generator.scss

```diff
@@ -0,0 +1,45 @@
+@use 'sass:map';
+@use 'sass:color';
+
+@function -adjust-lightness($color, $percentage: 10%) {
+	@if not $color or $color == transparent {
+		@return null;
+	}
+	$current-lightness: color.lightness($color);
+	$adjustment: if($current-lightness > 50%, -$percentage, $percentage);
+	@return color.adjust($color, $lightness: $adjustment);
+}
+
+@mixin generate-from($theme) {
+	.mat-mdc-slide-toggle {
+		$enabled-states: 'enabled', 'enabled:focus:not(:active)', 'enabled:active', 'enabled:hover:not(:focus):not(:active)';
+		$handle-selector: '.mdc-switch__handle::after';
+		$track-selector: '.mdc-switch__track';
+
+		$toggle: map.get($theme, 'general', 'toggle');
+		$handle-on: map.get($toggle, 'on');
```

Comment:
IMHO, we're not getting more readability (in this case) with $handle-on and $handle-off variables. I suggest merge them to $handle-on/off-background.